tools/forwarddiff-jl/run_hello.jl

```diff
@@ -0,0 +1,14 @@
+module Hello
+
+using ADTypes: AutoForwardDiff
+import ForwardDiff
+import GradBench
+
+GradBench.register!(
+    "hello", Dict(
+        "square" => GradBench.Hello.square,
+        "double" => (GradBench.Hello.double, AutoForwardDiff()),
```

Comment:
Can we instead just use a closure here?

Reply:
Not if you want to avoid the performance pitfalls of closures, e.g. with DI + Enzyme (which at the moment isn't measured but it would be unfair to set it up with a disadvantage)

Reply:
I don't quite understand? My propsal is essentially to define:

```
function double(x)
      GradBench.Hello.double(AutoForwardDiff(), x)
end
```

But with an anonymous function?